tests/test_sync.py

```diff
@@ -15,13 +15,15 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 """Test for sync.py module."""
-from unittest.mock import Mock
+import json
+from unittest.mock import MagicMock
```

Comment:
I think it is enough to only use `Mock`

```suggestion
from unittest.mock import Mock
```

Reply:
Done in 48d4b15.